magento/magento2
# Fixed conflicting styles for product image container in different listings at the same page

### Description (*)

This pull requests solves problem when there are two product listing blocks on the same page. Both blocks have at list one common product. And listings have different image sizes for product image.

### Related Pull Requests

Can't find related pull requests.

### Fixed Issues (if relevant)

Can't find related issue.

### Manual testing scenarios (*)

1. Open product "Crown Summit Backpack" (SKU 24-MB03) to assign up-sell products. Add at list one up-sell product. One of the assigned products must be "Fusion Backpack" (SKU 24-MB02). Save changes.
2. Create widget "Catalog Product List" via Magento Admin. Assign widget to store view you test. Layout updates settings: display on "Specified page"; page "Catalog Product View"; container "Main Content Top". Widget Options - select at least two product via SKU. One of products must be "Fusion Backpack" (SKU 24-MB02). Save widget.
3. Clear Magento 2 cache.
4. Open product page for "Crown Summit Backpack" (SKU 24-MB03) at store front.

![image](https://user-images.githubusercontent.com/13131588/103211368-af0b7480-4910-11eb-9169-a37f163e40ca.png)

With old code you can see that "Fusion Backpack" has smaller image than it should.

This pull request fixes it.  "Fusion Backpack" will get same size as "Affirm Water Bottle".

### Questions or comments


### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#31526: Fixed conflicting styles for product image container in different listings at the same page

### Resolved issues:
1. [x] resolves magento/magento2#35372: Fixed conflicting styles for product image container in different listings at the same page
